experimental/ExteriorAlgebra/src/ExteriorAlgebra.jl

```diff
@@ -193,3 +193,20 @@ end
 # # (1)  Computations with elements DO NOT AUTOMATICALLY REDUCE
 # #      modulo the squares of the generators.
 # # (2)  Do we want/need a special printing function?  (show/display)
+
+function Oscar.degrevlex(R::PBWAlgQuo)
+  return MonomialOrdering(R, Oscar.Orderings.SymbOrdering(:degrevlex, 1:nvars(R)))
+end
+
+function length(x::PBWAlgQuoElem)
+  return length(x.data.sdata)
+end
```

Comment:
These two functions don't belong in this file, but should instead be with the other PBWAlgQuo machinery